### PR TITLE
chore: add french translations to the changelog

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé

This PR adds [the french translations](https://docs.google.com/document/d/1DLsbKgTMzxyDpVoqnLlbsuzVO0pnN0Nw5Bws6QKjRRg/edit?tab=t.0) for the release notes on the changelog file

Also fixes the wrong values in the example

### CODE REVIEW

**Review by ethanWallace — APPROVED** [review]
LGTM

**Review by SmartMouthWords — COMMENTED** [review]
It looks like the typography changes section may be missing from the English. Maybe I'm not seeing it. it would be the following section: 
Typography changes
Only update if you use the caption value for the size property in the following components.
[gcds-icon](https://github.com/cds-snc/gcds-components/blob/main/packages/web/src/components/gcds-icon/readme.md)
Change caption to text-small
[gcds-text](https://github.com/cds-snc/gcds-components/blob/main/packages/web/src/components/gcds-text/readme.md)
Change caption to text-small

**Comment by daine** [comment]
@SmartMouthWords this PR only includes the fr text plus a small fix/change to the example provided in the english version. If you want to see the full version of the CHANGELOG that includes the french changes, here it is:
https://github.com/cds-snc/gcds-components/blob/docs/fr-translation/CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -592,6 +590,115 @@ Released on: 2024-02-22
...
+
+Version publiée le : 2024-09-25
+
+### :bug: :wrench: Corrections de bogues
+
+* Corriger les fautes de frappe dans les unités de style ([#646](https://github.com/cds-snc/gcds-components/issues/646)) ([4c8d850](https://github.com/cds-snc/gcds-components/commit/4c8d850569feafc538aa0e9593ba96cfdf9b97c2))
+* **gcds-sr-only:** Alignement avec la technique WCAG C7 ([#642](https://github.com/cds-snc/gcds-components/issues/642)) ([3cbe8bc](https://github.com/cds-snc/gcds-components/commit/3cbe8bc08d021849fa16269800fb08e12d4db037))
+* Mise à jour des unités de style pour corriger le rouge et s’aligner avec l’image de marque et corriger la taille de la police pour la légende du gcds-date-input ([#650](https://github.com/cds-snc/gcds-components/issues/650)) ([3c602da](https://github.com/cds-snc/gcds-components/commit/3c602dad949fcf16781cdef16be04c6870c94876))
+* Mise à jour des unités de styles pour propager les corrections du rouge de l’image de marque et la taille de la police de légende ([3c602da](https://github.com/cds-snc/gcds-components/commit/3c602dad949fcf16781cdef16be04c6870c94876))
```
[the remark this answers is not in the document]
Comment (SmartMouthWords) [reply CHANGELOG.md:679]: Yes, that's fine. I think the acronym FIP may be used more frequently in English but this way the French and English notes match.

[the remark this answers is not in the document]
Comment (daine) [reply CHANGELOG.md:679]:
```suggestion
* Mise à jour des unités de style pour corriger le rouge du PFIM et corriger la taille de la police pour la légende du gcds-date-input ([#650](https://github.com/cds-snc/gcds-components/issues/650)) ([3c602da](https://github.com/cds-snc/gcds-components/commit/3c602dad949fcf16781cdef16be04c6870c94876))
* Mise à jour des unités de styles pour propager les corrections du rouge du PFIM et la taille de la police de légende ([3c602da](https://github.com/cds-snc/gcds-components/commit/3c602dad949fcf16781cdef16be04c6870c94876))
```

@SmartMouthWords is this correct if we change it to `rouge du PFIM`? Or we can keep it the way it is and not bother with changing it.

[the remark this answers is not in the document]
Comment (daine) [reply CHANGELOG.md:679]:
```suggestion
* Mise à jour des unités de style pour corriger le rouge du PFIM (Programme fédéral de l’image de marque
) et corriger la taille de la police pour la légende du gcds-date-input ([#650](https://github.com/cds-snc/gcds-components/issues/650)) ([3c602da](https://github.com/cds-snc/gcds-components/commit/3c602dad949fcf16781cdef16be04c6870c94876))
```

**Review by SmartMouthWords — APPROVED** [review]

**Pull request merged**